src/renderer/state/flipper.ts

```diff
@@ -66,6 +112,10 @@ type FlipperMode =
   | 'front_disable'
   | 'back_enable'
   | 'back_disable'
+  | 'fr_disable'
```

Comment:
@IceSentry made me notice that we are not keeping the same name pattern : `fr_disable` vs `front_right_disable`.

Reply:
This would have to be changed marhkor_flippers.cpp aswell, since this is the name I'm using to call the ROS service. If we go ahead with this we need to make changes on markhor otherwise the services will no longer work.

Reply:
Alright, talk to @saxtot for this, it might be worth it to keep it coherent.